Library/Formula/tiger-vnc.rb

```diff
@@ -0,0 +1,26 @@
+class TigerVnc < Formula
+  homepage "http://tigervnc.org/"
+  url "https://github.com/TigerVNC/tigervnc/archive/v1.4.1.tar.gz"
+  sha1 "bdc5d6c5e55a275b645260e01c1e7aa71ba79984"
+
+  depends_on "cmake" => :build
+  depends_on "gnutls" => :recommended
+  depends_on "jpeg-turbo"
+  depends_on "gettext"
+  depends_on "fltk"
+
+  def install
+    turbo = Formula["jpeg-turbo"]
+    args = std_cmake_args + %W[
+      -DJPEG_INCLUDE_DIR=#{turbo.include}
+      -DJPEG_LIBRARY=#{turbo.lib}/libjpeg.dylib
+      .
+    ]
+    system "cmake", *args
+    system "make", "install"
+  end
+
+  test do
+    assert `#{bin}/vncviewer --help 2>&1` =~ /TigerVNC Viewer.*v1\.4\.1/
```

Comment:
Let's try to avoid hardcoding the version in the test, and use `assert_match pattern, string` instead.